lab-judy/app/component/gallery/edit-gallery/edit-gallery.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  template: require('./edit-gallery.html'),
+  controller: ['$log', 'galleryService', EditGalleryController],
+  controllerAs: 'editGalleryCtrl',
+  bindings: {
+    gallery: '<',
+  },
+};
+
+function EditGalleryController($log, galleryService){
+  $log.debug('init editGalleryCtrl');
+
+  this.updateGallery = function(){
+    galleryService.updateGallery(this.gallery._id, this.gallery);
```

Comment:
Good job using your bindings to set a gallery to the scope of this controller and use it to call your service method!